finesse/config.py

```diff
@@ -68,6 +68,14 @@
 OPUS_IP = "10.10.0.2"
 """The IP address of the machine running the OPUS software."""
 
+OPUS_POLLING_INTERVAL = 1.0
+"""How long to wait between polls of the EM27's status.
+
+Note that in reality the minimum poll interval is ~2s, because that's how long the
+device takes to reply. This value then determines how often we wait before even starting
+a request.
```

Comment:
I really don't understand this. Does it mean that the fastest we can get a response is every 3s?

Reply:
Sorry, this isn't v well worded. The spectrometer itself seems to handle incoming requests on a single thread and only responds to messages once every ~2s, so that's the max amount of time you'll be waiting.

We *could* just send a new status request as soon as we've received the last response and that would mean we'd still get an update every 2s, but it seemed sensible to limit how often we send requests from our side too (i.e. no more than one per sec), even if the result is the same in this case. If there was a new model that responded instantly to requests and we didn't have a limit then it would mean that we would spam it constantly.

Does that make sense?